dftimewolf/lib/module.py

```diff
@@ -169,8 +169,10 @@ def GetContainers(self,
 
     self.logger.debug(f'{self.name} is retrieving {len(containers)} '
         f'{container_class.CONTAINER_TYPE} containers - pop == {pop}')
-    for item in containers:
-      self.logger.debug(f'  * {str(item)} - origin: {item.src_module_name}')
+    for c in containers:
```

Comment:
I went with a single char variable because it either breaks up the f-string `{}` on 175 over multiple lines, or goes over the 80 character line length.